### PR TITLE
Add nano documentation for pytorch training quickstart

### LINKED ISSUE #4604: Nano: simplify the import line of pytorch trainer
We should import `bigdl.nano.pytorch.Trainer` here: https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/pytorch/__init__.py to enable `from bigdl.nano.pytorch import Trainer` or `bigdl.nano.pytorch.Trainer`

### PR DESCRIPTION
**Pull request by yangw1234**
Fixed #4604

### CODE REVIEW

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+We will briefly describe here the major features in BigDL-Nano for PyTorch training. You can find complete examples here [links to be added]().
+
+### Best Known Configurations
+
+When you run `source bigdl-nano-init`, BigDL-Nano will export a few environment variables, such as OMP_NUM_THREADS and KMP_AFFINITY, according to your current hardware. Empirically, these environment variables work best for most PyTorch applications. After setting these environment variables, you can just run your applications as usual (`python app.py`) and no additional changes are required.
+
+### BigDL-Nano PyTorch Trainer
+
+The PyTorch Trainer (`bigdl.nano.pytorch.Trainer`) is the place where we integrate most optimizations. It extends PyTorch Lightning's Trainer and has a few more parameters and methods specific to BigDL-Nano. The Trainer can be directly used to train a `LightningModule`.
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:15]: We should import `bigdl.nano.pytorch.Trainer` here: https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/pytorch/__init__.py

Comment (TheaperDeng) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:15]: raised an issue here: #4604

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+class MyModule(LightningModule):
+    #  LightningModule definition
+
+from bigdl.nano.pytorch import Trainer
+lightning_module = MyModule()
+trainer = Trainer(max_epoch=10)
+trainer.fit(lightning_module, train_loader)
+```
+
+For regular PyTorch modules, we also provide a "compile" method, that takes in a PyTorch module, an optimizer, and other PyTorch objects and "compiles" them into a `LightningModule`.
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:32]: should we make it clear what a "regular PyTorch module" is?

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+from bigdl.nano.pytorch import Trainer
+lightning_module = Trainer.compile(pytorch_module, optimizer, scheduler)
+trainer = Trainer(max_epoch=10)
+trainer.fit(lightning_module, train_loader)
+```
+
+#### Intel® Extension for PyTorch
+
+Intel Extension for Pytorch (a.k.a. IPEX) extends PyTorch with optimizations for an extra performance boost on Intel hardware. BigDL-Nano integrates IPEX through the `Trainer`. Users can turn on IPEX by setting `use_ipex=True`.
+
```
Comment (TheaperDeng) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:46]: IPEX is an optional dependency in bigdl-nano. Should we teach our users how to install ipex here?

Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:46]: I am thinking including ipex in setup.py after we upgrade ipex to 1.11

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md (hunk 1 above)
Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:15]: added the "import" in `bigdl.nano.pytorch`

**Comment by yangw1234** [comment]
http://10.112.231.51:18888/job/BigDL-NanoTests-PR-Validation/100/
